src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+{
+  "block.alloy_forgery.adamantite_block_forge_controller": "Adamantit Block Schmiede Kontrolleinheit",
+  "block.mythicmetals.adamantite_anvil": "Adamantit Amboss",
+  "block.mythicmetals.adamantite_block": "Adamantit Block",
+  "block.mythicmetals.adamantite_ore": "Adamantit Erz",
+  "block.mythicmetals.aquarium_anvil": "Aquarium Amboss",
+  "block.mythicmetals.aquarium_block": "Aquarium Block",
+  "block.mythicmetals.aquarium_glass": "Aquarium Glass",
+  "block.mythicmetals.aquarium_resonator": "Aquarium Resonator",
+  "block.mythicmetals.aquarium_steward": "Aquarium Steward",
+  "block.mythicmetals.aquarium_ore": "Aquarium Erz",
+  "block.mythicmetals.banglum_anvil": "Banglum Amboss",
+  "block.mythicmetals.banglum_block": "Banglum Block",
+  "block.mythicmetals.banglum_nuke_core": "Banglum Nuke Kern",
+  "block.mythicmetals.banglum_ore": "Banglum Erz",
+  "block.mythicmetals.banglum_tnt": "Banglum TNT",
+  "block.mythicmetals.blackstone_stormyx_ore": "Blackstone Stormyx Erz",
+  "block.mythicmetals.bronze_anvil": "Bronze Amboss",
+  "block.mythicmetals.bronze_block": "Bronze Block",
+  "block.mythicmetals.calcite_kyber_ore": "Calcite Kyber Erz",
+  "block.mythicmetals.calcite_starrite_ore": "Calcite Sternrit Erz",
+  "block.mythicmetals.carmot_anvil": "Carmot Amboss",
+  "block.mythicmetals.carmot_block": "Carmot Block",
+  "block.mythicmetals.carmot_nuke_core": "Carmot Nuke Kern",
```

Comment:
This should instead be something akin to "Carmotatomkern", or "Atomkern (Carmot)"

Reply:
I think it should be "Atombombenkern" because otherwise you'd just be talking about a singular nuclear core.